From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Javier Martinez Canillas <javierm@redhat.com>
Date: Thu, 27 Sep 2018 19:03:43 +0200
Subject: [PATCH] Include blscfg module for powerpc ieee1275

The blscfg module is currently not built for powerpc ieee1275, but this
is still needed when the machine is not booted in bare metal with OPAL.

Related: rhbz#1633646

Signed-off-by: Javier Martinez Canillas <javierm@redhat.com>
---
 grub-core/Makefile.core.def | 1 +
 1 file changed, 1 insertion(+)

diff --git a/grub-core/Makefile.core.def b/grub-core/Makefile.core.def
index f99fc994f6b..5b4841e1f00 100644
--- a/grub-core/Makefile.core.def
+++ b/grub-core/Makefile.core.def
@@ -782,6 +782,7 @@ module = {
   common = commands/blscfg.c;
   common = commands/bls_qsort.h;
   common = commands/loadenv.h;
+  enable = powerpc_ieee1275;
   enable = efi;
   enable = i386_pc;
   enable = emu;
